cellprofiler/__main__.py

```diff
@@ -576,6 +580,8 @@ def get_batch_commands(filename):
 
             return
 
+    print("Image numbers")
+    print(image_numbers)
```

Comment:
Yes, but I thought I added that to a different branch (I'm not that experienced with git).